app/src/main/java/com/framgia/fsalon/utils/formatter/CustomMarkerView.java

```diff
@@ -0,0 +1,43 @@
+
+package com.framgia.fsalon.utils.formatter;
+
+import android.content.Context;
+import android.widget.TextView;
+
+import com.framgia.fsalon.R;
+import com.github.mikephil.charting.components.MarkerView;
+import com.github.mikephil.charting.data.CandleEntry;
+import com.github.mikephil.charting.data.Entry;
+import com.github.mikephil.charting.highlight.Highlight;
+import com.github.mikephil.charting.utils.MPPointF;
+import com.github.mikephil.charting.utils.Utils;
+
+/**
+ * Custom implementation of the MarkerView.
+ *
+ * @author Philipp Jahoda
+ */
+public class CustomMarkerView extends MarkerView {
```

Comment:
WARNING: [ViewConstructor] Custom view `CustomMarkerView` is missing constructor used by tools: `(Context)` or `(Context,AttributeSet)` or `(Context,AttributeSet,int)`
 Some layout tools (such as the Android layout editor) need to find a constructor with one of the following signatures:
* `View(Context context)`
* `View(Context context, AttributeSet attrs)`
* `View(Context context, AttributeSet attrs, int defStyle)`

If your custom view needs to perform initialization which does not apply when used in a layout editor, you can surround the given code with a check to see if `View#isInEditMode()` is false, since that method will return `false` at runtime but true within a user interface editor.